app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -720,6 +720,8 @@ export class Dispatcher {
         baseBranch: baseBranch,
       })
     }
+
+    return result
```

Comment:
These two `return results` from the Dispatcher will help with handling state transitions inside the rebase flow.

Reply:
can we explicitly add the type for `results` to the function type definition? i'm fine with this pattern (even though we don't generally do it), I just want to make sure it's as clear as possible.

Reply:
Addressed in 238cf4fef